wallet/src/account_manager/operations/client.rs

```diff
@@ -159,6 +161,7 @@ impl AccountManager {
         let new_client = new_client_options.finish()?;
 
         let mut accounts = self.accounts.write().await;
+        #[allow(clippy::significant_drop_in_scrutinee)]
         for account in accounts.iter_mut() {
```

Comment:
Same here, why not
```suggestion
        #[allow(clippy::significant_drop_in_scrutinee)]
        for account in self.accounts.write().await.iter_mut() {
```
?